src/VAL_product_search/README.md

```diff
@@ -39,28 +39,50 @@ git clone --recurse-submodules https://github.com/alexa/amazon-alexa-language-as
 	- (Optional) GPU can be used for acceleration. 
 
 
-* Download the annotation dataset and gallery images according to the instructions in the challenge [website](https://eval.ai/web/challenges/challenge-page/1845/overview). To download the query images, download the gallery csv file (query\_file\_released.jsonl to ./src/VAL\_product\_search/datasets/data) and run following code in ./src
+* 2. Download the annotation dataset and gallery images according to the instructions in the challenge [website](https://eval.ai/web/challenges/challenge-page/1845/overview). 
+	 - To download the query images for training, download the training gallery csv file (train\_gallery\_file.csv to ./src/VAL\_product\_search/datasets/data) and run following code in ./src
+		```
+		python download_tool/data_download.py -o ./VAL_product_search/datasets/data/gallery_image -i ./VAL_product_search/datasets/data/train_gallery_file.csv -t gallery_csv
+		```
 
-	```
-	python download_tool/data_download.py -o ./VAL_product_search/datasets/data/product_search -i ./VAL_product_search/datasets/data/query_file_released.jsonl -t qe_jsonl
-	```
+	 - To download the query images for evaluation, download the evaluation gallery csv file (query\_file\_released.jsonl to ./src/VAL\_product\_search/datasets/data) and run following code in ./src
+		```
+		python download_tool/data_download.py -o ./VAL_product_search/datasets/data/product_search -i ./VAL_product_search/datasets/data/query_file_released.jsonl -t qe_jsonl
+		```
 
-* Download the trained VAL checkpoints from [Amazon Drive](https://www.amazon.com/clouddrive/share/TssaYesoP9ux2NpdEkorB0P7ikI7HyB1iLYcG3S8S6l), and put them under VAL\_product\_search/save\_model/product\_search/val\_resnet\_v2\_50\_ml/checkpoints/ directory.
+* 3. Download the trained VAL checkpoints (for evaluation) from [Amazon Drive](https://www.amazon.com/clouddrive/share/TssaYesoP9ux2NpdEkorB0P7ikI7HyB1iLYcG3S8S6l), and put them under VAL\_product\_search/save\_model/product\_search/val\_resnet\_v2\_50\_ml/checkpoints/ directory.
```

Comment:
Are you sure we don't need to change the name of the checkpoints here?

Reply:
I think we can keep the current names. But if to change, what names do you suggest?

Reply:
Changed as suggested, thanks!